model/config.go

```diff
@@ -1708,6 +1721,7 @@ type Config struct {
 	MessageExportSettings MessageExportSettings
 	JobSettings           JobSettings
 	PluginSettings        PluginSettings
+	SupportedTimezones    SupportedTimezones
```

Comment:
This feels like something that should either be hardcoded or grabbed from the OS, not something that anyone would actually configure

Reply:
It's pulling from a seperate file timezone.json so in a sense it is hardcoded.

pulling from OS would be problematic when you have the server on clusters.  Therefore, we took this apporach, of having a json for all timezones.

Something setting up the server does have the liberty to add only certain timezone.  Not sure why, but possible. Maybe all employees are in one location/region 🤷‍♂️

Reply:
I'm with @miguelespinoza on this, I think it's good to have it configurable if needed

Reply:
Sounds good

Reply:
Are these going to get written back to the config.json when that gets saved?